source/Core/Src/Settings.cpp

```diff
@@ -84,6 +84,7 @@ static const SettingConstants settingsConstants[(int)SettingsOptions::SettingsOp
     {0, 50, 1, 20},                                                 // PDNegTimeout
     {0, 1, 1, 0},                                                   // OLEDInversion
     {0, 99, 11, 33},                                                // OLEDBrightness
+    {0, 4, 1, 1},                                                   // LOGOTime
```

Comment:
Should we make this 0-5 or 0-10?

Reply:
I think 4 seconds is plenty.

And I will try to implement infinity mode as last option as initially proposed by you. Did not work yesterday, because I was not able to create an empty loop.

```cpp

while (buttons == BUTTON_NONE) {
}
```

Reply:
Finally managed to achieve this, but I need some help regarding the depiction of the infinity logo.